docs/Makefile

```diff
@@ -10,11 +10,24 @@ BUILDDIR      = _build
 
 # Put it first so that "make" without argument is like "make help".
 help:
+	ln -s _build/generated/authors.md
+	ln -s _build/generated/blackd.md
+	ln -s _build/generated/change_log.md
+	ln -s _build/generated/contributing_to_black.md
+	ln -s _build/generated/editor_integration.md
+	ln -s _build/generated/gnoring_unmodified_files.md
+	ln -s _build/generated/installation_and_usage.md
+	ln -s _build/generated/license.md
+	ln -s _build/generated/pyproject_toml.md
+	ln -s _build/generated/show_your_style.md
+	ln -s _build/generated/testimonials.md
+	ln -s _build/generated/the_black_code_style.md
+	ln -s _build/generated/version_control_integration.md
```

Comment:
`Makefile` is only used for Unix systems. Windows uses the `make.bat` file instead. I'll look into it and comment the Windows patch below once I have some free time.

Reply:
Edit: Actually with #1456, the symlinks were deleted, so all of this symlink nonsense can just be thrown in the garbage.

Reply:
Oh, I wasn't following up on that. Rebased to remove my no-op changes, thanks.